client/my-sites/post-type-list/post-actions-ellipsis-menu/duplicate.jsx

```diff
@@ -39,7 +39,7 @@ function PostActionsEllipsisMenuDuplicate( {
 	return (
 		<PopoverMenuItem href={ duplicateUrl } onClick={ onDuplicateClick } icon="clipboard">
 			<QueryJetpackModules siteId={ siteId } />
-			{ translate( 'Copy Post' ) }
+			{ translate( 'Copy post' ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.